mailpoet/assets/js/src/automation/listing/store/legacy-actions.tsx

```diff
@@ -67,8 +67,9 @@ export function* trashLegacyAutomation(automation: Automation) {
   createSuccessNotice(__('1 automation moved to the Trash.', 'mailpoet'));
```

Comment:
This is unrelated to your changes so feel free to ignore for now, but what do you think about updating these messages to include the name of the automation? Sometimes I move something to the trash but immediately forget which one I trashed. `n automation[s]` makes sense for bulk actions but when it's just 1 it can be nice to see the name.

Reply:
@johnolek Good point 👍  I added a few commits to fix this, unified the messages with the already-existing ones (such as `__('Automation "%s" was duplicated.', 'mailpoet')`).